lib/controller.js

```diff
@@ -937,20 +937,34 @@ controller.updateTesselWithVersion = function(opts, tessel, currentVersion, buil
     });
 };
 
-controller.tesselFirmwareVerion = function(opts) {
+controller.tesselFirmwareVerion = opts => {
   opts.authorized = true;
-  return controller.standardTesselCommand(opts, function(tessel) {
+  return controller.standardTesselCommand(opts, tessel => {
+    var tesselName = `Tessel [${tessel.name}]`;
+    var firmwareOutput = version => `${tesselName} Firmware version: ${version}`;
+    var nodeOutput = version => `${tesselName} Node version: ${version}`;
```

Comment:
I like it. Then `tesselName` could just be in the one function. Then it also paves a way for when #604 gets added.